### PR TITLE
[feature] Add a main script to print neuralprophet version

### PR DESCRIPTION
**Pull request by noxan**
Allows users to run `python -m neuralprophet --version` and receive the version information of the package directly from their terminal. Could be extended in the future to a cli application of neuralprophet.

### CODE REVIEW

**Review by ourownstory — CHANGES_REQUESTED** [review]
Nice addition!
Let's just use the long flag "--version".
Let's remove "-v", which is often used for verbose

**Comment by noxan** [comment]
@ourownstory Good point. Should we go with `-V` for version (capital letter) or fully remove the short version?

File: neuralprophet/__main__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Invokes neuralprophet when module is run as a script.
+"""
+import argparse
+
+from neuralprophet._version import __version__
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser(description="NeuralProphet")
+    parser.add_argument("-v", "--version", action="version", version="%(prog)s " + __version__)
```
Comment (noxan) [inline neuralprophet/__main__.py:11]: Remove `-v` option

```suggestion
    parser.add_argument("--version", action="version", version="%(prog)s " + __version__)
```

Comment (noxan) [inline neuralprophet/__main__.py:11]: Use capital letter `-V` for version shorthand

```suggestion
    parser.add_argument("-V", "--version", action="version", version="%(prog)s " + __version__)
```

**Review by karl-richter — APPROVED** [review]

**Pull request merged**